Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM ubuntu:xenial
+LABEL maintainer="Shaun Jackman <sjackman@gmail.com>"
+
+RUN apt-get update \
+	&& apt-get install -y --no-install-recommends bzip2 ca-certificates curl file fonts-dejavu-core g++ git locales make openssh-client patch sudo uuid-runtime \
+	&& rm -rf /var/lib/apt/lists/*
```

Comment:
What's this for? If cleaning: is there an `apt` command that can be used instead?

Reply:
This is what they recommend as a [best practice in the docker documentation](https://docs.docker.com/v17.09/engine/userguide/eng-image/dockerfile_best-practices/#run). I'm not sure if there's an `apt` command that can be used instead; I've always seen it like this.

Reply:
I would have expected `apt-get clean` to have an option to remove `/var/lib/apt/lists`, but I was not able to find such an option. See https://linux.die.net/man/8/apt-get